addons/mk6mortar/CfgVehicles.hpp

```diff
@@ -34,6 +35,59 @@ class CfgVehicles {
                 discreteDistanceInitIndex = 0;
             };
         };
+        class ACE_Actions: ACE_Actions {
+            class GVAR(unloadMagazine) {
+                displayName = CSTRING(unloadMortar);
+                distance = 2;
+                condition = QUOTE(_this call FUNC(canUnloadMagazine));
+                statement = QUOTE([ARR_3(_target,_player,5)] call FUNC(unloadMagazineTimer));
+                icon = "";
+                selection = "usti hlavne";
+            };
+            class GVAR(LoadActions) {
+                displayName = CSTRING(loadMortar);
+                distance = 2;
+                condition = QUOTE([ARR_2(_target,_player)] call FUNC(canLoadMagazine));
```

Comment:
The way it works right now is that if the mortar is loaded the unload action appears. If the mortar isn't loaded and you don't have any rounds, nothing appears. If the mortar isn't loaded and you have loadable rounds, the load actions appear.
I'm not entirely sure how to change anything with that.